pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -6607,6 +6607,570 @@
+  tutorial_pickcodechatbot_string_platforms: "All modern browsers, Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_codestersfish_string_platforms: "All modern browsers"
+  tutorial_codestersturtle_string_platforms: "All modern browsers"
+  tutorial_codestershedgehog_string_platforms: "All modern browsers"
+  tutorial_pickcodeggame_string_platforms: "All modern browsers"
+  tutorial_codestersdino_string_platforms: "All modern browsers"
+  tutorial_pickcodemtpaint_string_platforms: "All modern browsers"
+  tutorial_microsoftcarnival_string_platforms: "All modern browsers, Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_thunkablescavenger_string_platforms: "All modern browsers"
+  tutorial_codeillusionfund_string_platforms: "Chrome"
+  tutorial_codeillusiongoofybeg_string_platforms: "Chrome"
+  tutorial_codeillusiongoofyint_string_platforms: "Chrome"
+  tutorial_robotmagicescape_string_platforms: "All modern browsers, Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_codespeakclimate_string_platforms: "All modern browsers"
+  tutorial_codespeaktwinkle_string_platforms: "Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_coinbasecrypto_string_platforms: "All modern browsers, Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_codespeakcomp_string_platforms: "All modern browsers"
+  tutorial_codespeaksnake_string_platforms: "Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_prstemalexa_string_platforms: "All modern browsers"
+  tutorial_coderzfarm_string_platforms: "All modern browsers"
+  tutorial_intelinofield_string_platforms: "Unplugged"
+  tutorial_cablockchain_string_platforms: "Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_nasaparachute_string_platforms: "Unplugged"
+  tutorial_nasascratch_string_platforms: "Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_nasaorbit_string_platforms: "All modern browsers"
+  tutorial_nasapackage_string_platforms: "All modern browsers"
+  tutorial_nasalunar_string_platforms: "All modern browsers, iPad, iPhone"
+  tutorial_meeestate_string_platforms: "Android tablet, iPad, Windows phone, Android phone, iPhone"
+  tutorial_rpccw_string_detail_platforms: ""
+  tutorial_rpledfirefly_string_detail_platforms: "Raspberry Pi Pico"
+  tutorial_rpcharting_string_detail_platforms: ""
+  tutorial_rpsolar_string_detail_platforms: ""
+  tutorial_rpwebcards_string_detail_platforms: ""
+  tutorial_codeclubhworld_string_detail_platforms: ""
+  tutorial_codeclubspells_string_detail_platforms: ""
+  tutorial_coderdojoanime_string_detail_platforms: ""
+  tutorial_coderdojotarget_string_detail_platforms: ""
+  tutorial_creaticodescratch_string_detail_platforms: ""
+  tutorial_bbsierpinski_string_detail_platforms: ""
+  tutorial_bbstopwatch_string_detail_platforms: ""
+  tutorial_idrpatterns_string_detail_platforms: " "
```

Comment:
Should this be blank?

Reply:
Fixed. Yes, should be blank, but that extra space between quotes should not be there.